src/components/tab-title/tab-title.tsx

```diff
@@ -216,6 +218,7 @@ export class TabTitle implements InteractiveComponent {
     updateHostInteraction(this, () => {
       return this.selected;
     });
+    this.resizeObserver?.observe(this.containerEl);
```

Comment:
Can you move this to where you set the containerEl? `ref={(el) => (this.containerEl = el as HTMLDivElement)}`